.github/workflows/pr.yml

```diff
@@ -97,6 +97,26 @@ jobs:
         with:
           category: "Android Lint"
 
+  screenshot_test:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Looking at the flow diagram on #5158: wouldn't it make more sense to run this after lint/basic build has completed? So `    needs: [lint, lockfiles, ktlint]` like the other tests?

Reply:
yea grouped next to unit tests maybe?

Reply:
next to = running at the same time? That's what I was thinking, so basically as 'stage 3'.

Reply:
yup let me submit a PR to add that, thanks for the suggestion :)